packages/ui-components/src/components/buttons/IconButton.tsx

```diff
@@ -21,6 +22,7 @@ export const IconButton = ({
 				className={`text-${textColor} bg-${bgColor}-${disabled ? "300" : "400"} ${
 					!disabled && `hover:bg-${bgColor}-500`
 				} px-2 py-2`}
+				onClick={!disabled ? onClick : undefined}
```

Comment:
Probably instead of doing this we can pass the `disabled` param to the component?